metadata-ingestion/tests/integration/presto-on-hive/presto_on_hive_mces_golden_4.json

```diff
@@ -561,9 +561,9 @@
                 {
                     "com.linkedin.pegasus2avro.dataset.DatasetProperties": {
                         "customProperties": {
-                            "create_date": "2023-05-12",
-                            "table_type": "MANAGED_TABLE",
-                            "table_location": "hdfs://namenode:8020/user/hive/warehouse/db1.db/nested_struct_test"
+                            "create_date": "2023-07-05",
+                            "table_type": "EXTERNAL_TABLE",
+                            "table_location": "file:/opt/hive/data/warehouse/db1.db/nested_struct_test"
```

Comment:
these changes don't look right

Reply:
What do you expect?

Reply:
the `hdfs://` looks more like what I'd expect, not a local file path